### DISCUSSION TITLE
Immich Backup Guide Help

### DISCUSSION THREAD

**Discussion by Shoaib5136** [immich-app/immich#14311] [Q&A] [answered]
I’ve just set up Immich in an LXC container on Proxmox running Ubuntu, using Docker and Portainer. My question is about backups: I know that Immich automatically creates a backup file daily at 2 AM. If I copy all of its data, including the library DB data, and then upload it to a new instance, pasting the data there, will it work? Or is there a better method? What’s your preferred way to ensure the safety of your precious files in the long run?

![Screenshot 2024-11-24 134659](https://github.com/user-attachments/assets/cf0a09cf-fe95-4b8c-bcb9-3efc0e8c05da)
![Screenshot 2024-11-24 134815](https://github.com/user-attachments/assets/c5051992-513a-4d97-86b2-7ef5da683526)

**Comment by bo0tzz** [comment, marked as answer]
https://immich.app/docs/administration/backup-and-restore

**Reply by JeremyTBradshaw** [reply]
The backup instructions and every blog about them, all have the annoying catch 22 of if you don't stop the containers then the database backup and files backup might be out of sync, but if you stop the containers you can't do the database backup.

Never ever did I see anyone clear up this conundrum.  People just throw it out there that it doesn't matter and you just backup the files whenever, or they just pretend like it's all somehow possible without explanation.

The Immich backup and restore page stays the same.  Their example bash script that uses Borg doesn't say anything but obviously the containers stay running when they do that or the database backup command wouldn't be possible.

I wish they would clear this up.  A sound backup is like the #1 purpose of Immich 's existence.  Tons of people have asked.  Nobody ever got an answer.

If the containers don't need to be stopped then that note on the backup and restore page about database files being in sync should just be erased.

**Reply by bo0tzz** [reply]
> if you don't stop the containers then the database backup and files backup might be out of sync

https://docs.immich.app/administration/backup-and-restore/#backup-ordering addresses this

> but if you stop the containers you can't do the database backup.

You only need to stop immich-server, which is also mentioned in the link above.

**Reply by JeremyTBradshaw** [reply]
Thank you for pointing this out to me.  I was missing it.  Have now done `docker compose stop immich-server` and proceeded with the sample backup script's instructions.  Much appreciated.
